packages/xo-server/src/api/host.js

```diff
@@ -265,3 +265,19 @@ installSupplementalPack.params = {
 installSupplementalPack.resolve = {
   host: ['host', 'host', 'admin'],
 }
+
+// -------------------------------------------------------------------
+
+export function isHyperThreading({ host }) {
```

Comment:
`hyperThreadingEnabled`? Or `isHyperThreadingEnabled`?

Reply:
I will go for the second one :)

Reply:
I will go for the second one